docs-ref-conceptual/azure-cli-reference-for-devops.md

```diff
@@ -0,0 +1,151 @@
+
+### Azure Repos references
+
+[Azure Repos](/cli/azure/repos) has the following references:
+
+| Reference | Description | More information |
+|-|-|-|
+| [az repos](/cli/azure/repos) | Manage Azure Repos. | [What is Azure Repos?](/azure/devops/repos/get-started/what-is-repos) |
+| [az repos import](/cli/azure/repos/import) | Manage Git repositories import. | [Import a Git repo](/azure/devops/repos/git/import-git-repository) |
+| [az repos policy](/cli/azure/repos/policy) | Manage branch policies. | [Branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos policy approver-count](/cli/azure/repos/policy/approver-count) | Manage approver count policy. | [Branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos policy build](/cli/azure/repos/policy/build) | Manage build policies. | [Build validation](/azure/devops/repos/git/branch-policies#build-validation) |
+| [az repos policy case-enforcement](/cli/azure/repos/policy/case-enforcement) | Manage case-enforcement policy. | [Case enforcement](/azure/devops/repos/git/repository-settings#case-enforcement) |
+| [az repos policy comment-required](/cli/azure/repos/policy/comment-required) | Manage comment-required policy. | [Create, view, and manage pull requests](/azure/devops/repos/git/pull-requests) |
+| [az repos policy file-size](/cli/azure/repos/policy/file-size) | Manage file size policies. | [Improve code quality with branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos policy merge-strategy](/cli/azure/repos/policy/merge-strategy) | Manage merge strategy policy. | [Improve code quality with branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos policy required-reviewer](/cli/azure/repos/policy/required-reviewer) | Manage required reviewer policies. | [Improve code quality with branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos policy work-item-linking](/cli/azure/repos/policy/work-item-linking) | Manage work item linking policy. | [Improve code quality with branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos pr](/cli/azure/repos/pr) | Manage pull requests. | [Create, view, and manage pull requests](/azure/devops/repos/git/pull-requests) |
+| [az repos pr policy](/cli/azure/repos/pr/policy) | Manage pull request policies. | [Improve code quality with branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos pr reviewer](/cli/azure/repos/pr/reviewer) | Manage pull request reviewers. | [Improve code quality with branch policies](/azure/devops/repos/git/branch-policies) |
+| [az repos pr work-item](/cli/azure/repos/pr/work-item) | Manage work items associated with a pull request. | [Create, view, and manage pull requests](/azure/devops/repos/git/pull-requests) |
+| [az repos ref](/cli/azure/repos/ref) | Manage Git references. | [Manage branches](/azure/devops/repos/git/manage-your-branches) |
+
+## See also
```

Comment:
Staying consistent, rather than using "See also," most of our articles say, "Related articles."

Reply:
@chcomley, @dbradish-microsoft, the other summary reference articles also use "See also", so I think I should keep this one the same for consistency (but I could change all of them if Delora agrees).